Assets/SteamVR/Extras/SteamVR_LaserPointer.cs

```diff
@@ -91,11 +91,13 @@ public virtual void OnPointerOut(PointerEventArgs e)
 
         private void Update()
         {
-            if (!isActive)
+            if (active != isActive)
```

Comment:
Consider renaming active to state for a more logic naming.

Reply:
Dit script is niet van mij is van Valve